dist/ramda.js

```diff
@@ -990,8 +990,7 @@
     });
 
     /**
-     * Returns `true` if all elements of the list match the predicate, `false` if
-     * there are any that don't.
+     * Returns `false` if any element does not satisfy the predicate, `true` otherwise.
```

Comment:
I believe I was trying to say that the two sentences (red and green) are semantically equivalent.